index.html

```diff
@@ -179,6 +179,33 @@ <h5 class="btn-connect">LET’S CONNECT</h5>
                 </li>
             </ul>
     </section>
+    <footer class="footer-form">
```

Comment:
- [ ] Kindly you should try to stick to the design as much as possible to match the Figma design, so please work on the spacing between the form and about section to make it match the Figma design. you got this 👍 

Figma design:
![image](https://user-images.githubusercontent.com/90524466/186469776-4936cbb2-c579-4056-8596-c3c430300f60.png)

Your design:
![image](https://user-images.githubusercontent.com/90524466/186469918-d3a972fe-9bba-4a02-8c08-9177f33c00aa.png)